### ISSUE TITLE
[Visual Requirements – Make Code - Extensions] Content present on tiles are not properly visible after applying text spacing tool on 'Extensions' screen.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3693]
**User Experience**:
After running the text-spacing tool on the blade, then the text present on tiles are not properly visible due to which keyboard user failed to read the text of tiles.

**Note**: User credentials should NOT be included in the bug.

**Prerequisite:** 
Add below mentioned script in your browser and run the tool.

* {

     line-height: 1.5 !important;

     letter-spacing: 0.12em !important;

     word-spacing: 0.16em !important;
}

p {

     margin-bottom: 2em !important;
}

**URL:** https://chrome.google.com/webstore/detail/stylus/clngdbkpkpeebahjckkjfobafhncgmne?hl=en.

**Repro Steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Settings" option appearing in header section and select it.
6. Navigate to "Extensions" option and select it.
7. Navigate through all the controls appearing under "Extensions" page
8. Run the text spacing tool and verify, whether the text of tile is appearing properly or not.

**Actual Result:**
Content present on tiles are not properly visible after applying text spacing tool on 'Extensions' screen.

**Expected Result:**
Content present on tiles should be properly visible after applying text spacing tool on 'Extensions' screen.

Note: “if you believe this was caused by our use of a bookmarklet to test (e.g. the text highlighted in the bug is contained in an iframe), please comment and let us know.” Then the severity can be lowered to A11ySev3.

Below are the guidelines for Text spacing

This is specifically for text spacing bugs that are caused by an iframe, or other side effect of using a bookmarklet. If there are other text spacing issues, like text overlapping or breaking layout, those would be logged as usual.

Perhaps the best course of action would be to add some additional text to the bug template for all text spacing bugs that says something like “if you believe this was caused by our use of a bookmarklet to test (e.g. the text highlighted in the bug is contained in an iframe), please comment and let us know.” Then the severity can be lowered if that’s the case.

**WCAG Reference:**
[WCAG1.4.12 – Text Spacing](https://www.w3.org/WAI/WCAG21/Understanding/text-spacing.html)

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0)

**Comment by Shalini3457** [comment]
Labels:
#Accessibility; #A11ySev2; #HCL; #Benchmark-HCL-MakeCode-OCT2020; #WCAG1.4.12; #Win10-Edge(Chromium)

![WCAG1 4 12_Extensions_Text Spacing](https://user-images.githubusercontent.com/72871089/99038551-aa327180-25ab-11eb-80a1-5cf462987379.jpg)

**Comment by jwunderl** [comment]
These descriptions are optional preview descriptions / only shown when there is sufficient space (e.g. they are completely hidden on mobile devices, with the relevant information available through the 'learn more' dialog / documentation pages / extension source). We may want to reconsider this in the future if we do a larger redesign of the extensions page, but for now this is by design to limit to only a portion of the preview text.
